modules/extdata/uri/pom.xml

```diff
@@ -128,6 +220,32 @@
                     </dependency>
                 </dependencies>
                 <executions>
+                    <execution>
+                        <id>jar-badly-signed</id>
+                        <goals>
+                            <goal>run</goal>
+                        </goals>
+                        <phase>compile</phase>
+                        <configuration>
+                            <target>
+                                <mkdir dir="${basedir}/target/file_tmp/classes/org/apache/ignite/spi/deployment/uri/tasks/" />
+                                <copy file="${basedir}/target/classes/org/apache/ignite/spi/deployment/uri/tasks/GridUriDeploymentTestTask11.class" todir="${basedir}/target/file_tmp/classes/org/apache/ignite/spi/deployment/uri/tasks/" />
+                                <copy file="${basedir}/target/classes/org/apache/ignite/spi/deployment/uri/tasks/GridUriDeploymentTestTask11.class" tofile="${basedir}/target/file_tmp/classes/org/apache/ignite/spi/deployment/uri/tasks/GridUriDeploymentTestWithNameTask11.class" />
+
+                                <jar destfile="${basedir}/target/file/bad-signed-deployfile.jar" basedir="${basedir}/target/file_tmp/classes" />
+
+                                <signjar jar="${basedir}/target/file/bad-signed-deployfile.jar" keystore="${basedir}/config/signeddeploy/keystore" storepass="abc123" keypass="abc123" alias="business" />
+
+                                <sleep seconds="2" />
```

Comment:
Why?

Reply:
If JAR assembly and running this Ant script happens within one second, then the `zip` task won't detect any changes and leave the JAR not updated.
This `sleep` and `touch` bellow update the timestamp of the file, so the `zip` task doesn't ignore the file.
The same thing happens in the code bellow, where a similar GAR file is assembled.